presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/test/PlanBuilder.java

```diff
@@ -268,12 +269,17 @@ public TableScanNode tableScan(TableHandle tableHandle, List<Symbol> symbols, Ma
 
     public TableScanNode tableScan(TableHandle tableHandle, List<Symbol> symbols, Map<Symbol, ColumnHandle> assignments, Expression originalConstraint)
     {
+        return tableScanWithTableLayout(tableHandle, symbols, assignments, originalConstraint, Optional.empty());
+    }
+
+    public TableScanNode tableScanWithTableLayout(TableHandle tableHandle, List<Symbol> symbols, Map<Symbol, ColumnHandle> assignments, Expression originalConstraint, Optional<TableLayoutHandle> tableLayout)
```

Comment:
Do you need to use emphases `WithTableLayout`? It is already in parameter list, so everyone can see it.